source/Table/defaultHeaderRowRenderer.js

```diff
@@ -8,7 +8,7 @@ export default function defaultHeaderRowRenderer({
   style,
 }: HeaderRowRendererParams) {
   return (
-    <div className={className} role="row" style={style}>
+    <div className={className} role="rowheader" style={style}>
```

Comment:
`role="row"` was correct here. [`rowheader`](https://www.w3.org/TR/wai-aria/#rowheader) is for the cell that represents the heading for a row, if it exists.

Reply:
Oh my bad, bad understanding of the value. Thanks